src/olympia/reviews/views.py

```diff
@@ -432,5 +432,6 @@ def flag(self, request, *args, **kwargs):
         request = request._request
         response = flag(request, addon.slug, kwargs.get('pk'))
         if response.status_code == 200:
+            response.content = ''
```

Comment:
I'm re-using the legacy `flag` view here, because it already returns json, there was little point in duplicating it. But when it's successful, that function returns a `200` with an useless message as the content (`{"msg": "Thanks; this review has been flagged editor approval."}`), so I change it to a 202 (to indicate that nothing immediately happens when flagging). I had forgotten to remove the content until now.